certbot-ci/src/certbot_integration_tests/utils/certbot_call.py

```diff
@@ -135,7 +135,7 @@ def main() -> None:
 
     # Invoke certbot in test mode, without capturing output so users see directly the outcome.
     command, env = _prepare_args_env(args, directory_url, http_01_port, tls_alpn_01_port,
-                                     config_dir, workspace, True)
+                                     config_dir, workspace, False)
```

Comment:
I was scrolling through here because I was curious what structure you guys landed on -- I'm surprised this change doesn't make us skip any tests.